assets/src/edit-story/components/reorderable/reorderableScroller.js

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Can we move this inside `useReordering`?

Reply:
How? It's an independent component, no?

Reply:
But does it need to be? We can move the `onPointerEnter` and `onPointerLeave` inside `<Reorderable>` and handle the state and callbacks inside `useReordering`, no?

Reply:
The trick is, that it's a physical DOM element taking up the top and bottom x pixels of the component, and starts scrolling when hovered. If we wanted to do this based on the component itself, we'd need to calculate the position of the mouse in the parent container compared to the size of said container. It's of course possible, but requires a completely different approach to this.

Reply:
Yeah this is just a minor refactor to have everything be self contained inside the Reorderable component. What I meant is that we keep the same approach, just have it be inside `<Reaorderable/>` so instead of just `{children}` it would be `<ReorderableScroller />{children}<ReorderableScroller />` with `<ReorderableScroller />` being defined inside `<Reorderable />`

Reply:
Ah okay, I see what you mean. I've moved it to be internal to `reorderable` now, but kept it as a separate component defined outside just to make components smaller. See b52fb791.